react-components/src/components/RevealToolbar/SelectSingle3DModelButton.tsx

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type ReactElement } from 'react';
+
+import { useSDK } from '../RevealCanvas/SDKProvider';
+import { RevealContext } from '../RevealContext/RevealContext';
+import { type ModelWithRevision } from '../../hooks/types';
+import { Single3DModelSelection } from './Single3DModelSelection/SelectSingle3DModelSelection';
+
+type SelectSingle3DModelButtonProps = {
+  onSingleModelChanged?: (model: ModelWithRevision | undefined) => void;
+};
+
+export const SelectSingle3DModelButton = ({
+  onSingleModelChanged
+}: SelectSingle3DModelButtonProps): ReactElement => {
+  const sdk = useSDK();
+
+  const handleSingleModelChange = (model: ModelWithRevision | undefined): void => {
+    if (onSingleModelChanged !== undefined) onSingleModelChanged(model);
+  };
+
+  return (
+    <RevealContext sdk={sdk}>
```

Comment:
Why should the component be under `RevealContext`?
As `SelectSingle3DModelButton` is exported under `RevealToolbar`, I do not think it is necessary

Reply:
Hmmm strange. I only added because it was always complaining about without the context. Even under revealToolbar. Now is working without it. 🤔

Reply:
https://github.com/cognitedata/reveal/pull/4746/commits/cc1179ee0194008e281f93ecf79edbeb404ad2b1